### PR TITLE
RM-81673 Release over_react_codemod 1.8.0

### PR DESCRIPTION
**Pull request by rmconsole-readonly-wk**
Pull Requests included in release:
* Patch changes:
	* [More tweaks/improvements to boilerplate codemod](https://github.com/Workiva/over_react_codemod/pull/96)

Requested by: @greglittlefield-wf

Diff Between Last Tag and Proposed Release: https://github.com/Workiva/over_react_codemod/compare/1.7.0...Workiva:release_over_react_codemod_1.8.0
Diff Between Last Tag and New Tag: https://github.com/Workiva/over_react_codemod/compare/1.7.0...1.8.0

The logs for the request that created this PR can be found [here](https://w-rmconsole.appspot.com/release/release_event/6153708125028352/logs/)

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+## [1.8.0](https://github.com/Workiva/over_react_codemod/compare/1.8.0...1.7.0)
+
+Boilerplate codemod fixes and improvements:
+- Fix locally-generated semver reports not being readable due to not being nested inside an `{"exports": ...}` object
+- Treat FIXMEs converted to TODOs the same as FIXMEs (and don't add new FIXMEs on top of them)
+- Account for edge cases when Props were named `${componentName}ComponentProps`
+- Ignore common mixins that are known to not implement lifecycle methods and thus shouldn't require necessitate migration:
```
Comment (joebingham-wk) [inline CHANGELOG.md:7]:
```suggestion
- Ignore common mixins that are known to not implement lifecycle methods and thus shouldn't necessitate migration:
```

**Review by joebingham-wk — APPROVED** [review]
+10

**Comment by greglittlefield-wf** [comment]
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**